src/test/java/org/sonatype/cs/metrics/model/DbRowStrTest.java

```diff
@@ -0,0 +1,162 @@
+package org.sonatype.cs.metrics.model;
```

Comment:
I didn't catch this in our discussion, but I don't think we need to test getters and setters, assuming there is no actual logic in those getters and setters. Those functions are a  convention in Java. I think for this file, we need only to test the `toString()` override.

Reply:
I put the tests in because the code is handcrafted (so I trust it less). How about we move to using [Lombok](https://projectlombok.org/) to define these classes and then there's less risk and no need to test.

Reply:
I saw other things we could use to define them as well. However, IntelliJ has the capability to generate the code (not sure if you are familiar with the IDE), which is why I don't think we need testing. I would like to get at least one other person's opinion on it. Essentially, I don't want to over inflate the tests now if we don't have to.